test/chrome_test.js

```diff
@@ -47,9 +48,25 @@ describe("StorageArea", function() {
 
   /** @test {StorageArea#constructor} */
   describe("#constructor", function() {
-    it("should expose endpoint and areaName", function() {
+    it("should expose areaName", function() {
       expect(area.areaName).to.eql("sync");
-      expect(area.endpoint).to.eql("http://localhost:8080/v1/");
+    });
+  });
+
+  describe(".config setter", function() {
+    it("should set the config", function() {
+      const area = new StorageArea("sync");
+      area.config = { type: "kinto", remote: "http://localhost:8080/v1/" };
+      expect(area.config.remote).to.eql("http://localhost:8080/v1/");
+    });
+
+    it("should configure a sync timer", function(done) {
+      const area = new StorageArea("sync");
+      area.config = { type: "kinto", interval: 123000 };
+      setTimeout(function() {
```

Comment:
You should use sinon fake timers and sandbox.clock.tick() here http://sinonjs.org/docs/#clock